docs/testplan/Smartswitch-test-plan.md

```diff
@@ -0,0 +1,876 @@
+               VP5P0_CPU    5066 mV       5450      4550            5500           4500      False  20230619 11:31:11
+             VP5P0_SLED1    4964 mV       5250      4750            5350           4650      False  20230619 11:31:18
+             VP5P0_SLED2    4988 mV       5250      4750            5350           4650      False  20230619 11:31:27
+             VP5P0_SLED3    5003 mV       5250      4750            5350           4650      False  20230619 11:31:36
+             VP5P0_SLED4    5013 mV       5250      4750            5350           4650      False  20230619 11:31:46
+root@sonic:/home/cisco# 
+
+```
+#### Pass/Fail Criteria
+ * Verify warnings are all false.
+ * Verify changing the threshold values (high, low, critical high and critical low) and check alarm warning changing into True
+ * Verify changing back the threshold values to original one and check alarm warning changing into False
+
+
+### 1.3 Check platform temperature
+
+#### Steps
+ * Use command `show platform temperature` to get platform temperature
+
+#### Verify in
+ * Switch
+   
+#### Sample Output
+```
+On Switch:
+
+root@sonic:~#show platform temperature
+
+         Sensor    Temperature    High TH    Low TH    Crit High TH    Crit Low TH    Warning          Timestamp
+---------------  -------------  ---------  --------  --------------  -------------  ---------  -----------------
+        DPU_0_T         37.438      100.0      -5.0           105.0          -10.0      False  20230728 06:39:18
+        DPU_1_T         37.563      100.0      -5.0           105.0          -10.0      False  20230728 06:39:18
+        DPU_2_T           38.5      100.0      -5.0           105.0          -10.0      False  20230728 06:39:18
+        DPU_3_T         38.813      100.0      -5.0           105.0          -10.0      False  20230728 06:39:18
+     FAN_Sensor         23.201      100.0      -5.0           102.0          -10.0      False  20230728 06:39:18
+ MB_PORT_Sensor         21.813       97.0      -5.0           102.0          -10.0      False  20230728 06:39:18
+MB_TMP421_Local          26.25      135.0      -5.0           140.0          -10.0      False  20230728 06:39:18
+       SSD_Temp           40.0       80.0      -5.0            83.0          -10.0      False  20230728 06:39:18
+   X86_CORE_0_T           37.0      100.0      -5.0           105.0          -10.0      False  20230728 06:39:18
+   X86_PKG_TEMP           41.0      100.0      -5.0           105.0          -10.0      False  20230728 06:39:18
+```
+#### Pass/Fail Criteria
+ * Verify warnings are all false
+ * Verify changing the threshold values (high, low, critical high and critical low) and check alarm warning changing into True
+ * Verify changing back the threshold values to original one and check alarm warning changing into False
+
+
+### 1.4 Check DPU Console
+
+#### Steps
+ * Use command `/usr/bin/picocom -b 115200 /dev/ttyS<DPU_SLOT_NUM+OFFSET>` to access console for given dpu
```

Comment:
The serial device names may differ from vendor to vendor. We should include the mapping from DPU to the serial device in the platform.json file. The test should read the device information from platform.json.

Reply:
Changed it,